torchvision/models/efficientnet.py

```diff
@@ -651,13 +651,23 @@ class EfficientNet_V2_L_Weights(WeightsEnum):
 def efficientnet_b0(
     *, weights: Optional[EfficientNet_B0_Weights] = None, progress: bool = True, **kwargs: Any
 ) -> EfficientNet:
-    """
-    Constructs a EfficientNet B0 architecture from
-    `"EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks" <https://arxiv.org/abs/1905.11946>`_.
+    """EfficientNet B0 model architecture from the `"EfficientNet: Rethinking Model Scaling for Convolutional
+    Neural Networks" <https://arxiv.org/abs/1905.11946>`_ paper.
```

Comment:
here and in the rest of the docstrings, I think we can get rid of the quotes for a cleaner rendering

```suggestion
    """EfficientNet B0 model architecture from the `EfficientNet: Rethinking Model Scaling for Convolutional
    Neural Networks <https://arxiv.org/abs/1905.11946>`_ paper.
```